platform/src/main/java/org/stellar/anchor/platform/event/ClientStatusCallbackHandler.java

```diff
@@ -17,8 +29,44 @@ public ClientStatusCallbackHandler(ClientConfig clientConfig) {
 
   @Override
   void handleEvent(AnchorEvent event) {
-    // TOOD: Implement calling the business server
+    // initialize the http client
+    OkHttpClient client = new OkHttpClient.Builder()
+            .connectTimeout(10, TimeUnit.MINUTES)
+            .readTimeout(10, TimeUnit.MINUTES)
+            .writeTimeout(10, TimeUnit.MINUTES)
+            .callTimeout(10, TimeUnit.MINUTES)
+            .build();
+
+    // Prepare the payload to sign
+    String currentTs = String.valueOf(TimeUnit.MILLISECONDS.toSeconds(System.currentTimeMillis()));
+    String domain;
+    try {
+      domain = getDomainFromURL(clientConfig.getCallbackUrl());
+    } catch (MalformedURLException e) {
+      throw new RuntimeException(e);
```

Comment:
Do we do validation on the callback URL?

Reply:
We don't. `callback_url` is optional (see comments [here](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/platform/src/main/resources/config/anchor-config-default-values.yaml#L155)). But will make change to the callsite to make sure `callback` is present

Reply:
Maybe we should do validation then? I think it's a bad experience to find out the URL is malformed only during runtime. Doesn't have to be in this PR.

Reply:
The callback_url in config is optional (see comments [here](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/platform/src/main/resources/config/anchor-config-default-values.yaml#L155), and disussion [here](https://github.com/stellar/java-stellar-anchor-sdk/pull/970#discussion_r1251134903)). I have change the callsite to check if callback_url is empty.

Validating callback_url in format won't help here because the exception thrown here is the result of `getDomainFromURL()` trying to create a `URL` object from string, we will need that try catch block anyways

Reply:
Hey Jiahui, if `callback_url` is not empty (although it is optional), we should check the URL format in the [ClientsConfig ](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/platform/src/main/java/org/stellar/anchor/platform/config/ClientsConfig.java#L42)validator.